hasher-matcher-actioner/webapp/src/Api.tsx

```diff
@@ -324,23 +324,23 @@ export async function fetchHashCount(): Promise<Response> {
 
 // TODO remove the trailing slash from the API URL, then add back the leading slash for /actions/ and /action-rules/ endpoints.
 
+// This class should be kept in sync with python class ActionPerformer
```

Comment:
nit/unsure: include file path

Reply:
I think thats more likely to change than the class name. Changing the class name you would see the comment saying these should be kept in sync

Reply:
haha the python file comments in this class disagree :P 
I would include both because then it is easy to trace the history of changes if it is missed.